### ISSUE TITLE
Media path failure

### ISSUE THREAD

**Issue by authoritimmy** [kit-ty-kate/mastodon-archive-viewer#1]
When attempting to generate an archive, the following error occurs
```
[tim@yoga13 Downloads]$ mastodon-archive-viewer outbox.json > archive.html
mastodon-archive-viewer: internal error, uncaught exception:
                         FFmpeg.Avutil.Failure("Failed to open input /horse-media/media_attachments/files/000/003/042/original/40b72a4002ba4cfc.mp4")
```
The path listed doesn't exist on my system. If I exclude media-posts and media-replies it works fine. I created a simlink to make it a valid path on my system and things worked just fine.

This is on Manjaro. I extracted the archive directly to my ~/Downloads folder and ran the command as shown in the above code block. There was no folder called horse-media anywhere, and certainly none at my root directory

**Comment by kit-ty-kate** [comment]
arf, sounds like mastodon changed its archive format. I'll fix that shortly

**Comment by authoritimmy** [comment]
It also was pointing at my root directory instead of the current working directory.

**Comment by kit-ty-kate** [comment]
Could you try with the latest master? It should support Mastodon 2.6.

Though looking again at your error it seems to be a different problem indeed :/ Which mastodon version are you using? And also any forks or patches?

**Comment by kit-ty-kate** [comment]
I added an explicit version detection in https://github.com/kit-ty-kate/mastodon-archive-viewer/commit/b59bb4fb19780e7d63e4e2d9cc32e0027fb8ad1b. If your instance produces different archive setup tell me.

**Comment by authoritimmy** [comment]
I uninstalled fully, then git cloned it again and reinstalled (this would get me the latest master, right?) Both versions are referred to as 0.3~, but I'm pretty sure I updated it.  

The instance I downloaded the archive from is running 2.6.1. I don't run the instance so I'm not really sure of any tweaks. I also attempted it with an older archive from bofa.lol (RIP). i extracted to ~/bofa and it looked for a /media_attachements folder at root (without the horse-media level folder in the path, interestingly).

I deleted the symlinks I used as a workaround and both archives worked correctly on the newest version. Seems to be fixed. Thank you for looking into this
